UnityProject/Assets/Scripts/Drawers/Morgue.cs

```diff
@@ -155,7 +157,8 @@ private IEnumerator PlayAlarm()
 		alarmRunning = true;
 		while (consciousnessPresent && buzzerEnabled && !alarmBroken)
 		{
-			SoundManager.PlayNetworkedAtPos("OutOfAmmoAlarm", DrawerWorldPosition, sourceObj: gameObject);
+			// JESTER
+			//SoundManager.PlayNetworkedAtPos("OutOfAmmoAlarm", DrawerWorldPosition, sourceObj: gameObject);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)